Maddori.Apple/Maddori.Apple/Screen/MyReflection/MyReflectionDetail/MyReflectionDetailViewController.swift

```diff
@@ -140,21 +141,37 @@ final class MyReflectionDetailViewController: BaseViewController {
 extension MyReflectionDetailViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         // FIXME
-        return contentArray.count
+        if contentArray.isEmpty {
+            return 1
+        } else {
+            return contentArray.count
+        }
```

Comment:
p5;

```suggestion
        return contentArray.isEmpty ? 1 : contentArray.count
```

한 줄로 바꿀 수 있을 것 같군요 !

Reply:
[61c06b4](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/146/commits/61c06b49bd225258e5bf26a62c3a38eb3be4938e)
대박적